Добавление плащей для отделов, новых балаклав для людей и новых военных курточек для гражданских!

Exapsters opened the pull request:
<!--
**Это перевод оригинального темплейта с Baystation12/Baystation12**
Не забудьте добавить ченджлог, если вы сделали изменения, касающиеся админов/игроков, и которые могут повлиять на геймплей.

Примеры, когда необходимо добавить ченджлог:
* Добавление/удаление объектов с которыми игроки могут взаимодействовать, или их поведение.
* Добавление/удаление/изменение админских инструментов/кнопок.
* Изменения карты.

Примеры, когда записи в ченджлоге необязательны/не требуются:
* Косметические изменения, такие как описания, звуки и т.д.
* Оптимизация и другие подобные изменения базовых систем, не влияющие на геймплей.
* Небольшие багфиксы.

Вы можете найти README файл с подробными объяснениями в ./html/changelogs.
А ещё можно добавить свой ченджлог прямо в ПР. Инфа тут: https://github.com/Proxima-Project/Proxi-Bay12/wiki/Automatic-changelog-generation
-->
### Описание изменений
Можно начать с того, что некоторые просили добавить меня плащи... и я это сделал! Сверх этого доступны новые балаклавы и курточки, потому скучный вид сделается менее скучным.
<!--
Здесь вы пишите, какой прекрасный йогурт вы сделали.
-->
### Почему и что этот ПР улучшит
Сделает скучный вид гражданских персонажей чуть менее скучным.
<!--
Здесь вы пишите, почему ваш йогурт лучше чем тот, что был.
-->
### Авторство
Часть кода была портирована с Infitity Project.
<!--
Здесь вы пишите, если этот йогурт был украден у сырков.
-->
### Чейнджлог
:cl: Exapster
imageadd: Добавлены новые военные куртки для гражданских в лодаут.
imageadd: Добавлены разноцветные балаклавы в лодаут для ОСБ. Доступы исключительно расе людей.
imageadd: Добавлены плащи в лодаут для ОП, МБ, НБ, ОО, ОСБ. Доступны только ветви гражданских.
imageadd: Отныне можно носить плащи и нарукавные повязки на любых скафандрах.
rscdel: Удален здравый смысл.
/:cl:
<!--
Здесь вы пишите, если о новом йогурте должны узнать игроки. Пример смотрите в шапке
-->
#### Требуемые изменения на википедии проекта
Требуемые изменения отсутствуют.
<!--
Здесь требуется описание всех изменений, что могут, потенциально, затрагивать лор или геймплейную составляющую игры, ввиду чего требуется обновление википедии.
Если вы не уверены в том, требуются ли они и какие именно - обратитесь к Директору Редакторума в нашем Дискорд сервере (https://discord.gg/F7GgD86Pvf)
Вот наша основная вики: https://wiki.proxima.fun/wiki/Заглавная_страница
-->

Exapsters committed:
RE01

Время летит...

Evie7056 commented on maps/torch/loadout/loadout_accessories.dm:
```diff
@@ -190,6 +190,59 @@
+	display_name = "cloak, security"
+	path = /obj/item/clothing/accessory/cloak/security
+	allowed_branches = CIVILIAN_BRANCHES
+	allowed_roles = SECURITY_ROLES
```
В итоге их смогут носить только дефективы. Гениально.

Evie7056 commented:
У вас насрано в ДМЕ и аксессуарах. Слишком доисторическая ветка.

Exapsters commented:
> У вас насрано в ДМЕ и аксессуарах. Слишком доисторическая ветка.

Ж-е-с-т-ь.

Evie7056 commented on code/modules/clothing/spacesuits/spacesuits.dm:
```diff
@@ -138,7 +138,8 @@
 	species_restricted = list("exclude", SPECIES_NABBER, SPECIES_DIONA)
-	valid_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA)
+	valid_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, \
+								ACCESSORY_SLOT_ARMBAND) // PRX@CODE
```
Дерьмокод раз.

Evie7056 commented on proxima/code/modules/clothing/accessories/cloak.dm:
```diff
@@ -0,0 +1,67 @@
+	species_restricted = null
+	siemens_coefficient = 0.9
+	w_class = ITEM_SIZE_NORMAL
+	slot = ACCESSORY_SLOT_ARMBAND
```
Дерьмокод два.

Evie7056 commented:
Не доделано.

Exapsters committed:
Попытка исправить файл с акс

Exapsters committed:
RE

Некий нехороший человек положил туда один пренеприятнейший файл, а страдать после этого мне.

4310V343k commented:
насри скринов

Exapsters commented:
![изображение](https://user-images.githubusercontent.com/98767708/181263286-71898d04-cff3-4846-9de1-7f3d53189c15.png)

Exapsters commented:
![изображение](https://user-images.githubusercontent.com/98767708/181263363-ce9240f3-6b76-4dfd-892f-28dac5c9a3dd.png)

Exapsters committed:
Merge remote-tracking branch 'upstream/dev' into Ex-Cloak-Mask-Mil

Mazzik12 commented:
![image](https://user-images.githubusercontent.com/42623478/181268335-26b96c2d-d261-46a0-bf44-3e4ed42622c4.png)
POV: 
![image](https://user-images.githubusercontent.com/42623478/181268372-0a83478f-ac98-4498-8fc0-f09fba2dbc31.png)

Exapsters committed:
REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Evie7056 commented on code/modules/clothing/spacesuits/spacesuits.dm:
```diff
@@ -139,7 +139,8 @@
 	randpixel = 0
 	species_restricted = list("exclude", SPECIES_NABBER, SPECIES_DIONA)
-	valid_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA)
+	valid_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, \
```
Убери слеш, он тут не нужен.

Exapsters replied:
> Убери слеш, он тут не нужен.

Ошибочка, да.

Evie7056 commented on maps/torch/loadout/loadout_head.dm:
```diff
@@ -117,6 +117,12 @@
+/datum/gear/tactical/balaclava/humanized
+	display_name = "balaclava - human's special"
+	path = /obj/item/clothing/mask/balaclava/humanized
+	flags = GEAR_HAS_SUBTYPE_SELECTION
```
Хз что это за флаг, но я не думаю от него проблем будете.

Exapsters replied:
> Хз что это за флаг, но я не думаю от него проблем будете.

Этот флаг позволяет выбирать из списка.

Evie7056 commented on proxima/code/modules/clothing/masks/masks.dm:
```diff
@@ -0,0 +1,50 @@
+		slot_wear_mask_str = 'proxima/icons/mob/onmob/onmob_mask.dmi'
+	)
+	species_restricted = list(
+		SPECIES_HUMAN
```
Насрано. Подвиды людей и Шеллов не смогут это надевать.

polerno replied:
> Насрано. Подвиды людей и Шеллов не смогут это надевать.

А так и надо, покажем что люди лучше всех остальных

Evie7056 replied:
Дурак что ли?

Exapsters replied:
> Дурак что ли?

Какие ещё добавить?

Evie7056 replied:
> > Дурак что ли?
> 
> Какие ещё добавить?

Шеллы, Ваты, Спейс и грав-адапты. Триторианцы по желанию. Они же жирные

Evie7056 replied:
Флаги не подскажу, смотри их сам

Exapsters committed:
Merge branch 'dev' into Ex-Cloak-Mask-Mil

Exapsters committed:
Merge branch 'dev' into Ex-Cloak-Mask-Mil

Exapsters committed:
Re?

Exapsters commented:
Я сделал это за мятный пряник, потому мне есть оправдание.

Evie7056 commented on proxima/code/modules/clothing/masks/masks.dm:
```diff
@@ -0,0 +1,50 @@
+		slot_wear_mask_str = 'proxima/icons/mob/onmob/onmob_mask.dmi'
+	)
+	species_restricted = list(
+		HUMAN_SPECIES
```
Экзапстер блять, ты еблан, у ватгроунов и остальных ебало одинаковое, хватит вставать в позу и сделай нормально

Evie7056 replied:
Стоп. Подожди. Кажется, это я еблан.

Evie7056 commented:
Я еблан.
